draft-demarco-oauth-status-assertions.md

```diff
@@ -154,6 +157,11 @@ to prove the non-revocation status of the digital credential to a verifier.
 ~~~
 **Figure 2**: Status Assertion Presentation Flow.
 
+In summary, the credential issuer provides the client with a
+Status Assertion, which is linked to a Digital Credential. This enables
+the client to present both the digital credential and its
+Status Assertion to a verifier as proof of the digital credential's
+non-revocation status.
```

Comment:
```suggestion
current status.
```

Reply:
This of course assumes that status information is not limited to revocation.

Reply:
Maybe it is better to say validity status instead of current status. The Status Assertion only gives a validity proof.